### ISSUE TITLE
Возможно ли сделать что бы работали и поиск ссылок на использование функции?

### ISSUE THREAD

**Issue by arXXXimed** [1c-syntax/vsc-language-1c-bsl#189]
Работает стандартная команда на переход на определение процедуры или функции ( Go to Definition [F12])
Планируется ли использование переход на использование процедур 
(Go to reference[shift + F12])?

**Comment by asosnoviy** [comment]
По мере появления этого  в languageServer

**Comment by nixel2007** [comment]
Go to reference и find reference не одно и то же?

**Comment by asosnoviy** [comment]
Не, "показать определение" и "показать все места использования" же.

**Comment by nixel2007** [comment]
Так, а то, что показывается в ховере над вызовом функции с зажатым ctrl или shift (или alt, не помню) - это оно?

**Comment by arXXXimed** [comment]
> Так, а то, что показывается в ховере над вызовом функции с зажатым ctrl или shift (или alt, не помню) - это оно?

Нет..

**Comment by nixel2007** [comment]
Окей, тогда как должно выглядеть то, что вы хотите?

**Comment by arXXXimed** [comment]
> Окей, тогда как должно выглядеть то, что вы хотите?

Так же, как это работает со всеми остальными языками программирования... например на Python
![2B7B2BD4-9EB3-483F-B9B1-669074474867](https://user-images.githubusercontent.com/36421796/74651504-02f7e980-5195-11ea-9370-d59d506ad39e.GIF)

**Comment by nixel2007** [comment]
![image](https://user-images.githubusercontent.com/1132840/74652020-2c654500-5196-11ea-8d94-4e9d3415519a.png)

Оно?

**Comment by arXXXimed** [comment]
ХМ.. я такого результата добиться не смог (((

**Comment by nixel2007** [comment]
VSCode открыл в режиме каталога или отдельного файла?

**Comment by arXXXimed** [comment]
> ![image](https://user-images.githubusercontent.com/1132840/74652020-2c654500-5196-11ea-8d94-4e9d3415519a.png)
> 
> Оно?

Да . получается так.. Раширение как то странно работает при в ключенном или отключенном languige Server
При включенном Reference не работают , при выключенном работают...
Причем изменения отключения вступают в силу только после переоткрытия VScode

**Comment by arXXXimed** [comment]
Ну и работает только на сохраненных файлах... в новых ( что бы вставить и быстро проанализировать текст) не работает ((

**Comment by arXXXimed** [comment]
я прошу прощения. Оно то работает , то не работает , 
То  работает только через горячие клавиши, а только потом начинает работать через контекстное меню... Пока закономерности не обнаружил
